ansible/roles_ocp_workloads/ocp4_workload_rhte19_shadowman/tasks/workload.yml

```diff
@@ -0,0 +1,137 @@
+---
+- name: Setting up workload for user
+  debug:
+    msg: "Setting up workload for user ocp_username = {{ ocp_username }}"
+
+# Kafka
+- name: Create Project  
+  shell: oc new-project {{ocp_project}}
+
+- name: Create Cluster Operator Role
+  shell: oc adm policy add-cluster-role-to-user strimzi-cluster-operator-namespaced --serviceaccount strimzi-cluster-operator -n {{ocp_project}}
```

Comment:
Same. `k8s` module please. If you need an example of a YAML for a (Cluster) role binding: https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles_ocp_workloads/ocp4_workload_rhtr_pipelines_user/templates/rolebinding.yaml.j2